### PR TITLE
Setup react-app

### PR DESCRIPTION
**Pull request by koachcode**
## Setup react app
1. setup react app using a create-react-app calculator
2. No linter errors
3. Used Gitflow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
...
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (hiromataba) [inline README.md]: - [ ] Please update your README file to make your project professional, You may have a look at [This Template](https://github.com/microverseinc/readme-template) Kindly remove the default `CRA` template and update your README file

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi :wave: :wave: @Ginohmk 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :medal_military: 
- Correct gitflow :heavy_check_mark: 
- Your linters are passing :+1: 

## Required Changes ♻️



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please remove the icons and images in your public directory generated by `CRA` because you are not going to use them so let's kindly remove them

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (koachcode) [reply README.md]: Thank you for your feedback, I will work on it.

**Review by dasileker — APPROVED** [review]
## **Hi @Ginohmk,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**